scss/elements/_forms.scss

```diff
@@ -177,6 +177,12 @@ input[type="file"] {
   }
 }
 
+// set smaller width for checkbox label
```

Comment:
I never explained this, but all of the `/elements/` partials should be unclassed, html element selectors only. So let's move this style to the `_dropdowns.scss` partial.